styleguide.config.js

```diff
@@ -179,6 +179,33 @@ module.exports = {
                 'react/components/EXPERIMENTAL_Charts/BubbleChart/index.tsx',
               ],
             },
+            {
+              name: 'Table V2',
+              content:
+                'react/components/EXPERIMENTAL_Table/docs/Introduction.md',
+              sections: [
+                {
+                  name: 'Features',
+                  content:
+                    'react/components/EXPERIMENTAL_Table/docs/Features.md',
+                },
+                {
+                  name: 'Components',
+                  content:
+                    'react/components/EXPERIMENTAL_Table/docs/Components.md',
+                },
+                {
+                  name: '🧪E2E Testing',
```

Comment:
I'd drop the emojies but that's really my opinion. Great job otherwise.

Reply:
I'll do that. Thanks :)